docs/installation/index.md

```diff
@@ -1,7 +1,16 @@
 # Installation
 
-* Download and install Java 1.8, then set an environment variable: `export JAVA_HOME=$(/usr/libexec/java_home -v 1.8)`
-* Get Spark 2.3.x: [Download](https://spark.apache.org/downloads.html), unzip it and then set an environment variable: `export SPARK_HOME=<SPARK_FOLDER>`
+* Download and install Java 1.8
+    * To stay sane, some prefer to use JENV to manage different versions of Java: [Recommended Blog](https://medium.com/@danielnenkov/multiple-jdk-versions-on-mac-os-x-with-jenv-5ea5522ddc9b)
+    * On a Mac you might install Java 1.8 with: `brew cask install adoptopenjdk/openjdk/adoptopenjdk8`
+* Determine which Java version you are using (See JENV above):
+  * `$ which java`
+  * `/Users/<user>/.jenv/shims/java`
+* Set the JAVA_HOME environment variable.  Examples: 
+  * `export JAVA_HOME=$(/usr/libexec/java_home -v 1.8)`
+  * `export JAVA_HOME=$(/Users/<user>/.jenv/shims/java -v 1.8)`
+* Get Spark 2.4.x: [Download](https://spark.apache.org/downloads.html), unzip it and then set an environment variable: `export SPARK_HOME=<SPARK_FOLDER>`
+    * some users also found this useful: [Recommended Blog](https://www.tutorialkart.com/apache-spark/how-to-install-spark-on-mac-os/)
```

Comment:
We should only expose one way of installing software. Too many options can be confusing. I would remove this link.

Reply:
And I am sure they become out of date easily . . .